setup.py

```diff
@@ -1,5 +1,6 @@
 #! /usr/bin/env python
 
+# Copyright (C) 2008-2016 The scikit-sparse developers
```

Comment:
I forgot the remove the old copyright entries. I have done it now. So it is just the following.
```
Copyright (C) 2008-2016 The scikit-sparse developers
```
I think this is the best way. Or what do you think?